### PR TITLE
Give Alchemy mechanic Spellbook Draft its own API 

### PR DESCRIPTION
**Pull request by Northmoc**
Previously I had hacked it together using NameCard + MakeCard and it was just messy, especially considering how to support the Hourglass Coven.

This should be better, I think.

Initial testing seems just fine.

### CODE REVIEW

**Comment by Hanmac** [comment]
Draft Effect needs TriggerZoneAll stuff with ChangesZoneTable

**Comment by Hanmac** [comment]
Some Spellbook cards does Conjure instead of Draft, we should probably handle that too

**Comment by Northmoc** [comment]
> Draft Effect needs TriggerZoneAll stuff with ChangesZoneTable

I think MakeCardEffect also needs that!

File: forge-game/src/main/java/forge/game/ability/effects/DraftEffect.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                 game.getAction().moveTo(ZoneType.None, card, sa, moveParams);
+                 drafted.add(card);
+             }
+         }
+
+         final CardZoneTable triggerList = new CardZoneTable();
+         for (final Card c : drafted) {
+             game.getAction().moveTo(zone, c, sa, moveParams);
+             if (c != null) {
+                 triggerList.put(ZoneType.None, c.getZone().getZoneType(), c);
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/DraftEffect.java:77]: Should probably use the card from moveTo for these

**Review by tool4ever — APPROVED** [review]

**Pull request merged**